tests/kata-containerized-test-tool/cmd/katatest/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/kata-containers/tests/kata-containerized-test-tool/internal/core"
+	"github.com/kata-containers/tests/kata-containerized-test-tool/internal/tests/cpu"
+	"github.com/kata-containers/tests/kata-containerized-test-tool/internal/tests/memory"
+)
+
+func main() {
+	outputDir := flag.String("output", "results", "Output directory for test results")
+	flag.Parse()
+
+	if err := os.MkdirAll(*outputDir, 0755); err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to create output directory: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Create the framework
+	framework := core.NewFramework(*outputDir)
+
+	// Register all available tests
+	framework.RegisterTest(cpu.New())
+	framework.RegisterTest(memory.New())
```

Comment:
I would extract this list of tests to its own file, this way it's trivial to modify.